db/dataset.py

```diff
@@ -160,6 +160,62 @@ def get(id):
             return None
 
 
+def get_public_datasets(status_value, offset, limit):
+    result = ''
+    with db.engine.connect() as connection:
+        if status_value == "all":
+            query = """
+            SELECT dataset.id :: TEXT
+                 , dataset.name
+                 , dataset.description
+                 , dataset.author
+                 , "user".musicbrainz_id AS author_name
+                 , dataset.created
+                 , job.status
+            FROM dataset
+            JOIN "user"
+              ON "user".id = dataset.author
+              LEFT JOIN LATERAL (SELECT status
```

Comment:
I didn't know AB used JSONB. In that case, ignore my comment (I see you've
already updated INSTALL.md).